src/Core/src/Platform/Android/MauiSwipeRefreshLayout.cs

```diff
@@ -20,6 +20,11 @@ public class MauiSwipeRefreshLayout : SwipeRefreshLayout
 
 		public MauiSwipeRefreshLayout(Context context) : base(context)
 		{
+			// This works around a bug in SwipeRefreshLayout
+			// https://github.com/dotnet/maui/pull/17647#discussion_r1433358418
+			// https://issuetracker.google.com/issues/110463864
+			// It looks like this issue is fixed on the main branch of Android but it hasn't made its way into the packages yet
+			SetProgressViewOffset(true, ProgressViewStartOffset, ProgressViewEndOffset - Math.Abs(ProgressViewStartOffset));
```

Comment:
The problem is, is that I have no idea what version of AndroidX this will actually be released on. 

It's fixed on the main branch of their code but I don't really know how to track what version of AndoridX it will show up in.